tests/core/partitioning/test_fallback_graph_output.cpp

```diff
@@ -7,99 +7,110 @@
-  auto trt_results = trt_mod.forward(trt_inputs_ivalues).toTensor();
-  ASSERT_TRUE(torch_tensorrt::tests::util::almostEqual(jit_results, trt_results, 2e-6));
-}
-
-TEST(Partitioning, ComputeMobileNetFallbackGraphCorrectly) {
-  torch::jit::script::Module mod;
-  try {
-    mod = torch::jit::load("tests/modules/mobilenet_v2_traced.jit.pt");
-  } catch (const c10::Error& e) {
-    std::cerr << "error loading the model\n";
-    return;
-  }
-
-  const std::vector<std::vector<int64_t>> input_shapes = {{1, 3, 224, 224}};
-  std::vector<torch::jit::IValue> jit_inputs_ivalues;
-  std::vector<torch::jit::IValue> trt_inputs_ivalues;
-  for (auto in_shape : input_shapes) {
-    auto in = at::randint(5, in_shape, {at::kCUDA});
-    jit_inputs_ivalues.push_back(in.clone());
-    trt_inputs_ivalues.push_back(in.clone());
-  }
-
-  std::vector<torch_tensorrt::core::ir::Input> input_ranges{torch_tensorrt::core::ir::Input({1, 3, 224, 224})};
-  auto g = mod.get_method("forward").graph();
-  torch_tensorrt::core::CompileSpec cfg(input_ranges);
-  cfg.partition_info.enabled = true;
-  cfg.partition_info.forced_fallback_operators.push_back("aten::hardtanh");
-
-  auto jit_results = mod.forward(jit_inputs_ivalues).toTensor();
-  auto trt_mod = torch_tensorrt::core::CompileGraph(mod, cfg);
-  auto trt_results = trt_mod.forward(trt_inputs_ivalues).toTensor();
-  ASSERT_TRUE(torch_tensorrt::tests::util::almostEqual(jit_results, trt_results, 2e-6));
-}
-
-TEST(Partitioning, ComputeResNet50HalfFallbackGraphCorrectly) {
-  torch::jit::script::Module mod;
-  try {
-    mod = torch::jit::load("tests/modules/resnet50_traced.jit.pt");
-  } catch (const c10::Error& e) {
-    std::cerr << "error loading the model\n";
-    return;
-  }
-
-  mod.to(torch::kHalf);
-
-  const std::vector<std::vector<int64_t>> input_shapes = {{1, 3, 224, 224}};
-  std::vector<torch::jit::IValue> jit_inputs_ivalues;
-  std::vector<torch::jit::IValue> trt_inputs_ivalues;
-  for (auto in_shape : input_shapes) {
-    auto in = at::randint(5, in_shape, {at::kCUDA}).to(torch::kHalf);
-    jit_inputs_ivalues.push_back(in.clone());
-    trt_inputs_ivalues.push_back(in.clone());
-  }
-
-  auto in_shape = torch_tensorrt::core::ir::Input({1, 3, 224, 224});
-  in_shape.dtype = nvinfer1::DataType::kHALF;
-
-  std::vector<torch_tensorrt::core::ir::Input> input_ranges({in_shape});
-  auto g = mod.get_method("forward").graph();
-  torch_tensorrt::core::CompileSpec cfg(input_ranges);
-  cfg.partition_info.enabled = true;
-  cfg.partition_info.forced_fallback_operators.push_back("aten::add");
-
-  auto jit_results = mod.forward(jit_inputs_ivalues).toTensor();
-  auto trt_mod = torch_tensorrt::core::CompileGraph(mod, cfg);
-  auto trt_results = trt_mod.forward(trt_inputs_ivalues).toTensor();
-  // Lower threshold because FP16
-  ASSERT_TRUE(torch_tensorrt::tests::util::almostEqual(jit_results, trt_results, 2e-1));
-}
+// TEST(Partitioning, ComputeResNet50FallbackGraphCorrectly) {
```

Comment:
@bowang007 Can you work with @peri044 here to determine the best testing strategy here?

Reply:
sure.